components/web/firefox/patches/Bug577056.patc

```diff
@@ -0,0 +1,50 @@
+diff --git a/components/web/firefox/firefox-68.12.0/js/mapfile.IllumOS b/js/mapfile.IllumOS
```

Comment:
This file is also unused - is it intentional?

Reply:
yes I will remove these two files. This was the attempt to get the tagged pointer things running.